de.json

```diff
@@ -1941,65 +1941,65 @@
+  "obd_pid_loading": "Lade die Fahrzeug-OBD-PIDs...",
+  "obd_pid_loading_failed": "Pids konnten nicht abgerufen werden!",
   "obd_pid_test_pids": "Test Pids",
-  "obd_pid_delete_title": "Delete custom OBD Pids",
-  "obd_pid_delete_description": "Are you sure you want to delete your custom OBD Pids?",
-  "obd_pid_virtual_pid": "Virtual Pid",
-  "obd_pid_virtual_pid_description": "Virtual Pids use other telemetry keys to calculate a value, e.g. power is often not provided by cars but can be calculated by using voltage and current equation results",
+  "obd_pid_delete_title": "Benutzerdefinierte OBD-Pids löschen",
+  "obd_pid_delete_description": "Bist du sicher, dass du deine benutzerdefinierten OBD-Pids löschen möchtest?",
+  "obd_pid_virtual_pid": "Virtuelles Pid",
+  "obd_pid_virtual_pid_description": "Virtuelle Pids verwenden andere Telemetrieschlüssel, um einen Wert zu berechnen, z.B. wird die Leistung oft nicht von den Fahrzeugen geliefert, kann aber anhand der Ergebnisse der Spannungs- und Stromwerte berechnet werden.",
   "obd_pid_protocol_generic": "Generic",
-  "obd_pid_data": "Raw OBD response",
-  "obd_pid_parsed_response": "Parsed TLM data",
-  "obd_pid_log": "Log",
+  "obd_pid_data": "Unbearbeitete OBD-Antwort",
+  "obd_pid_parsed_response": "Geparste TLM-Daten",
+  "obd_pid_log": "Protokoll",
   "obd_pid_equation_map": "Map",
-  "obd_pid_new_equation_map_entry": "New Entry",
-  "obd_pid_equation_map_description": "This maps certain equation results to the specified value, use \"default\" to match all equation results you did not specify. Values must be numeric, due to keyboard restrictions on different systems we show the fully featured keybard.",
-  "obd_pid_equation_map_count_zero": "No entries",
-  "obd_pid_equation_map_count_one": "One entry",
-  "obd_pid_equation_map_count_other": "{{count}} entries",
-  "obd_pid_share_verify_first": "Please test your pids first before sharing, thank you!",
-  "obd_pid_share_completed": "Thank you for sharing your pids! We'll review them and make sure everyone benefits from your improvements.",
-  "idle": "Idle",
-  "started": "Started",
-  "succeeded": "Succeeded",
-  "failed": "Failed",
-  "unnecessary": "Unnecessary",
-  "rivian_vehicles_in_account": "These are your Rivian vehicle(s). Would you like to add these to your ABRP account and connect live data?",
-  "rivian_add_vehicles": "Add these vehicle(s)",
-  "skip_for_now": "Skip for now",
-  "abrpobd_connect_failed_tip": "It can help to remove the dongle from the car and reinsert it."
-}
\ No newline at end of file
+  "obd_pid_new_equation_map_entry": "Neuer Eintrag",
+  "obd_pid_equation_map_description": "Dies ordnet bestimmte Entsprechungsergebnisse dem angegebenen Wert zu. Verwende \„default\“, um alle Entsprechungsergebnisse zuzuordnen, die du nicht angegeben hast. Die Werte müssen numerisch sein, aufgrund von Tastaturbeschränkungen auf verschiedenen Systemen zeigen wir die voll ausgestattete Tastatur.",
```

Comment:
```suggestion
  "obd_pid_equation_map_description": "Dies ordnet bestimmte Entsprechungsergebnisse dem angegebenen Wert zu. Verwende „default“, um alle Entsprechungsergebnisse zuzuordnen, die du nicht angegeben hast. Die Werte müssen numerisch sein, aufgrund von Tastaturbeschränkungen auf verschiedenen Systemen zeigen wir die voll ausgestattete Tastatur.",
```

Reply:
Nur " braucht ein Escape.

Reply:
Im englischen Original steht an dieser Stelle [use \\"default\\" to match all equation results](https://github.com/Iternio-Planning-AB/abrp-translations/blob/35eb1e0bf78fa71dc4a5daca52e0cebdc5ebb9cd/en.json#L1963).
Darum habe ich die \\" Zeichen so belassen.